docs/5-development/07-ts.md

```diff
@@ -71,19 +65,35 @@ class MyClass extends UI5Element {
 }
 ```
 
-### Defining the tag and other settings
+### Defining the tag, static getters and other settings
 
-Use `@customElement` as a replacement for the metadata `tag` setting.
+Use `@customElement` as a replacement for the metadata `tag` setting and all the `static get` methods.
 
-Use `@themeAware`, `@languageAware` and `@fastNavigation` where needed. Note that these do not accept parameters.
+Use `themeAware`, `languageAware` and `fastNavigation` where needed.
+
+**Example**
 
-Example:
 ```ts
-@customElement("my-element-name")
-@languageAware
-@themeAware
-@fastNavigation
+@customElement({
```

Comment:
`renderer` missing. I know this isn't an exhaustive example, however if you have template, you must have renderer, and vice versa - if you don't have a template, renderer is redundant. So better to make the example correct.